models/containerized_model/templates/prediction_service_h2o_mojo.py

```diff
@@ -39,27 +30,51 @@ def read_and_save_file(source_path, destination_path):
     with open(destination_path, 'wb') as f:
         f.write(model_binary)
 
-# <Fill in this function and remove the 'raise'>
-def _get_prediction_class(preds):
-    """
-    For a classification model, _get_prediction_class should return the
-    appropriate class label based on the class probability.
-
-    For example, for a binary classification model:
-        if preds[0] > preds[1]:
-            return 1
-        return 0
-    """
-    raise Exception('_get_prediction_class not implemented')
-
-
 
 class MojoModelWrapper(SimpleModelWrapper):
 
+    def __get_columns(self):
+        """
+        user-defined helper method. all user-defined methods must be prefixed by __
+        `__get_columns` is user defined helper method to list column names in order to re-create the dataframe for prediction
+        **Note**: all user defined methods must be declared inside the class as `instance methods`
+                  no user defined methods are allowed outside the scope of the class instantiating the `SimpleModelWrapper`
+                  for e.g. all methods must be declared inside the `GermanCredit` class (in this case) and referenced using `self`
+
+        :return: list of column names
+        """
+        columns = []
+        if not len(columns) == 0:
+            return columns
+        raise Exception('Columns can not be empty. Please update columns in prediction_service.py file.')
+
+    def set_global_imports(self):
+        """
+        overridden method to make external global imports
+        When using external imports override the method to provide necessary imports. Make sure to mark them `global` to
+        be used by certifai interpreter correctly. set once,use throughout.
+        :return: None
+        """
+        global dt
+        import datatable as dt
+
+    # <Fill in this function and remove the 'raise'>
+    def _get_prediction_class(self, preds):
```

Comment:
It's highly desirable to have a "will usually work for H2O" implementation here, and have the user override it only if needed using a list of the outcome classes in the order they will be returned (c.f. columns).   We can then move the overrides into metadata that's optionally provided with the model but that'll need more refactoring of containerization scripts so perhaps leave that out of this PR.

Also think we should rename the function to be _get_prediction given what it does in the regression case is return a probability not the class

Reply:
all these are 1.3.6. like gunicorn won't work with 1.3.5  if any of the models require external imports like xgb and datatable (in-case of h20). I can add that docs specifying these examples work with 1.3.5 and we should keep this in a separate branch for now and merge as and when required

Reply:
_get_prediction for regression returns actual prediction values like a floating point  number not probabilities, so yea i should rename it to get_prediction

Reply:
h2o that works out of box will require us to either fix deafult case as binary classification or accept metadata file

Reply:
@akumar-cs Why can't we figure it out from the preds (1, 2 or more results per...)?